### PR TITLE
feat: improve commons-js coverage

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | -- |
| Dependencies | -- |
| Decisions | - Improve code coverage to ripe-commons-js - Not yet at 100%, missing CSV File reader functions and Auth module, since these ones require somehow mocking browser FileReader and mocking/using ripe-id with token, which will take more time. Since the code changes are already big, I'm opening the PR now as is and the missing parts will come later |
| Animated GIF | -- |

### CODE REVIEW

**Comment by BeeMargarida** [comment]
CI failing due to eslint, but there is already a PR with the fix https://github.com/ripe-tech/ripe-commons-js/pull/40

File: test/express/handlers.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                {},
+                res,
+                next
+            );
+            assert.strictEqual(errorResult.statusCode, "400");
+            assert.strictEqual(errorResult.code, "400");
+            assert.strictEqual(errorResult.error, "Error");
+            assert.strictEqual(errorResult.stack, undefined);
+
+            process.env.NODE_ENV = "development";
```
Comment (joao-conde) [inline test/express/handlers.js:107]: Maybe a try catch to restore previous node env value

**Review by joao-conde — CHANGES_REQUESTED** [review]

**Comment by joao-conde** [comment]
@BeeMargarida add the coveralls badge to the readme maybe, once @joamag  activates it in coveralls app I think it will show

File: test/express/handlers.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                    {},
+                    res,
+                    next
+                );
+                assert.strictEqual(errorResult.statusCode, "400");
+                assert.strictEqual(errorResult.code, "400");
+                assert.strictEqual(errorResult.error, "Error");
+                assert.strictEqual(errorResult.stack, undefined);
+            } finally {
+                process.env.NODE_ENV = "development";
```
Comment (joao-conde) [inline test/express/handlers.js:108]: Should we assume the previous was development? Maybe save the previous value like

```js
const prev = process.env.NODE_ENV;
process.env.NODE_ENV = "production";
....
finally {
  process.env.NODE_ENV = prev;
}
```

**Pull request merged**